### DISCUSSION TITLE
A question about inverse problem

### DISCUSSION THREAD

**Discussion by Lastilla** [lululxvi/deepxde#1066] [Q&A]
@lululxvi 
I`m so sorry to bother you. I have some doubt about inverse problem.  In PINN, we set some points as boundary condition:
```
    [ob_x, ob_y, ob_u, ob_v, ob_p, ob_theta] = database()
       
    ob_xy = np.hstack((ob_x, ob_y))
    
    observe_u = dde.icbc.PointSetBC(ob_xy, ob_u, component=0)
    observe_v = dde.icbc.PointSetBC(ob_xy, ob_v, component=1)
```
but after trainning, we need input points of the same number we input. 
I input 10 points to predict, and model would output 10 points, just like I tell librarian the book index, and he give the book I need. 
This question puzzles me for a long time.

**Comment by lululxvi** [comment]
I cannot understand your question.

**Comment by Lastilla** [comment]
sry for my poor interpretation.&nbsp; I mean that: in prediction phase, can I rebuild the flow fluid through several points?



---Original---
From: "Lu ***@***.***&gt;
Date: Mon, Dec 12, 2022 04:56 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [lululxvi/deepxde] A question about inverse problem (Discussion#1066)




 
I cannot understand your question.

**Comment by lululxvi** [comment]
You can predict the u/p at any point.
